git/git.go

```diff
@@ -307,6 +307,15 @@ func CheckoutBranch(branch string) error {
 	return run.PrepareCmd(configCmd).Run()
 }
 
+//pull changes from remote branch without version history
+func PullLatestChanges() error {
+	pullCmd, err := GitCommand("pull", "--ff-only")
```

Comment:
I think the pull command should be supplied with remote and branch name as arguments to avoid pulling absolutely everything from the repo.